riak_test/tests/gc_tests.erl

```diff
@@ -206,3 +223,47 @@ verify_riak_object_remaining_for_bad_key(RiakNodes, GCKey, {{Bucket, Key}, UUID}
     lager:info("As expected, BAD manifest in GC bucket remains,"
                " stand off orphan manfiests/blocks: ~p", [Manifest]),
     ok.
+
+verify_partial_gc_run(CSNode, RiakNodes,
+                      {MegaSec0, Sec0, _},
+                      {MegaSec1, Sec1, _}) ->
+    Start0 = MegaSec0 * 1000000 + Sec0,
+    End0 = MegaSec1 * 1000000 + Sec1,
+    Interval = erlang:max(1, (End0 - Start0) div 3),
+    Starts = [ {Start0 + N * Interval, Start0 + (N+1) * Interval}
+               || N <- lists:seq(0, 3) ],
```

Comment:
It is apparent it takes more than 8 seconds, as the object deletion process above is like this and each has 2 seconds interval:

```erlang
    [begin
         setup_normal_obj([{"hop", 42}, {"step", 65536}, {"jump", 7}], UserConfig),
         timer:sleep(2000)
     end || _ <- lists:seq(0,3) ],
```

So by dividing 8 by 3, `Starts` is like {0, 2}, {2, 4}, {4, 6} and {6, 8}. That's why I put `lists:seq(0,3)`, Not `(0, 2)`.